packages/client/src/getBalances.ts

```diff
@@ -36,7 +45,7 @@ export const getBalances = async (evmAddress: any, btcAddress = null) => {
         zrc20: token.zrc20_contract_address,
       });
       tokens.push({
-        chain_id: 7001,
+        chain_id: this.network === "mainnet" ? 7000 : 7001,
```

Comment:
I've switched to using a helper to get ZetaChain's chain ID from the client to avoid hard-coding IDs.

```
chain_id: this.getChainId(`zeta_${this.network}`)
```

3a61455